<launch>
  <node name="serial_stream_node" pkg="serial_stream" type="serial_stream_node" output="screen">
    <param name="serial_port" value="/dev/teensy"/>
    <param name="baud_rate" value="115200"/>
    <param name="data_count" value="1"/>
    <param name="data0_format" value="freq: %lf"/>
    <remap from="~data0" to="frequency"/>
  </node>

  <include file="$(find leptrino_force_torque)/launch/leptrino.launch" />
</launch>